### PR TITLE
Support submitted by charts

### LINKED ISSUE #851: Add support for _submitted_by to charts endpoint
The form _submitted_by meta field is not supported by the charts endpoint, unlike the _submission_time.

### PR DESCRIPTION
**Pull request by denniswambua**
closes #851

### CODE REVIEW

File: onadata/libs/utils/chart_tools.py [hunk 1]
```diff
@@ -208,6 +208,10 @@ def build_chart_data_for_field(xform, field, language_index=0, choices=None,
         field_label = 'Submission Time'
         field_xpath = '_submission_time'
         field_type = 'datetime'
+    elif isinstance(field, basestring) and field == common_tags.SUBMITTED_BY:
```
Comment (pld) [inline onadata/libs/utils/chart_tools.py]: why check that this is a string?

Comment (ukanga) [inline onadata/libs/utils/chart_tools.py]: I think this is a pattern already laid out before, see most of the if statements already in the file, also a field tends to be an object most of the time.

Comment (pld) [inline onadata/libs/utils/chart_tools.py]: can we remove it? if it's an object this would fail in the 2nd conjunct, why's the first part of this needed?

**Comment by pld** [comment]
@denniswambua we're also going to want to add `_duration`, is that included already, if not can you do that too?

The conditional blocks have a repeating pattern, it looks like there's probably a better way to abstract this.

**Comment by denniswambua** [comment]
`_duration` has not been implemented let me add it.

**Pull request merged**